### ISSUE TITLE
Why no web?

### ISSUE THREAD

**Issue by asbjornenge** [sindresorhus/anatine#13]
I would :heart: to use on mobile :iphone: :stuck_out_tongue_winking_eye: :star:

Let me rephrase that; "What is the reasoning behind using electron for this?" If you don't mind me asking :smile:

**Comment by fvsch** [comment]
Anatine is a way to use the new mobile.twitter.com on desktop operating systems. It's based on this Chrome extension from the same author:
https://github.com/sindresorhus/refined-twitter

Part of the need for this extension and Anatine (which is basically Chrome + this extension bundled as an app) is that mobile.twitter.com:
- might redirect users to twitter.com if they are on big screens or send the “wrong” User-Agent
- serves the older mobile.twitter.com site to some User-Agents (e.g. Firefox).

If you'd like to use https://mobile.twitter.com on mobile, try loading it in your mobile browser. It will still filter on User-Agent, so Firefox on Android gets the old version, and apparently Chrome on Android gets the new one.

**Comment by asbjornenge** [comment]
@fvsch aha! Thanks for clearing that up for me :smile: :+1:
